great_expectations/jupyter_ux/expectation_explorer.py

```diff
@@ -1939,7 +1939,7 @@ def generate_expectation_suite_editor_widgets(self, data_asset, expectation_suit
         data_asset_state = self.state["data_assets"].get(data_asset_name, {})
         data_asset_expectations = data_asset_state.get("expectations", {})
 
-        ge_version = expectation_suite.get("meta")["great_expectations.__version__"]
+        ge_version = expectation_suite.get("meta")["great_expectations_version"]
```

Comment:
Mark with a todo to return and fix later, but for now let's accept both versions so this isn't breaking.